modules/menuItems.js

```diff
@@ -601,7 +601,7 @@ let menuTempl = function(webviews) {
         label: i18n.t('mist.applicationMenu.develop.syncModeLight'),
         enabled:
           ethereumNode.isOwnNode &&
-          ethereumNode.isGeth &&
+          store.getState().nodes.local.client === 'geth' &&
```

Comment:
I believe the ` === geth ` is no longer necessary, as this line would only be parsed after `ethereumNode.isOwnNode` returns true. And we do not have another own client anymore.